client.py

```diff
@@ -0,0 +1,20 @@
+import socket
+import sys
+
+from kvstore.constants import INPUT_PROMPT
+
+def main():
+    while True:
+        i = input(INPUT_PROMPT)
+
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
not sure if i should be creating a new socket for each input

Reply:
No, generally better to use the same socket whenever possible (ie when connected to the same host). Imagine you have a web server and database server, and that the database server can handle 50 concurrent connections... best configuration is to have a socket pool of 50 sockets, and reuse them as needed across all web requests. Biggest thing this helps with is latency, as you avoid the TCP and TLS handshakes.